Source/Meadow.Foundation.Core/Displays/DisplayBase.cs

```diff
@@ -1,88 +1,66 @@
 ﻿using Meadow.Foundation.Graphics;
 using Meadow.Foundation.Graphics.Buffers;
-using Meadow.Peripherals.Displays;
+using System;
 
 namespace Meadow.Foundation.Displays
 {
     /// <summary>
     /// Abstract hardware display class 
     /// </summary>
+    [Obsolete]
```

Comment:
Could we add a string message with a reason of why its obsolete, or encourage to use what instead?

Reply:
done!